jobs/vsphere_cpi/templates/cpi.erb

```diff
@@ -20,15 +20,7 @@ export NO_PROXY="<%= no_proxy %>"
 export no_proxy="<%= no_proxy %>"
 <% end %>
 
-<% if_p('vcenter.connection_options.ca_cert') do %>
-  export BOSH_CA_CERT_FILE=$BOSH_JOBS_DIR/vsphere_cpi/config/cacert.pem
-<% end %>
-<% if_p('vcenter.nsx.ca_cert') do %>
-  export BOSH_NSX_CA_CERT_FILE=$BOSH_JOBS_DIR/vsphere_cpi/config/nsx_cacert.pem
-<% end %>
-<% if p('vcenter.nsxt.ca_cert', false) %>
-  export BOSH_NSXT_CA_CERT_FILE=$BOSH_JOBS_DIR/vsphere_cpi/config/nsxt_cacert.pem
-<% else %>
+<% if !p('vcenter.nsxt.ca_cert', false) %>
   export NSXT_SKIP_SSL_VERIFY=true
```

Comment:
Is NSXT_SKIP_SSL_VERIFY not also something that should be determined by the config passed by the director?

Reply:
Good catch, we're running the integration tests with this environment variable removed. Will resolve this comment once they go green.

Reply:
Tests went green w/ the removal.